packages/oauth/src/index.spec.js

```diff
@@ -447,6 +447,7 @@ describe('OAuth', async () => {
           assert.fail('should have failed');
         },
         failure: async (error, installOptions, req, res) => {
+          assert.isDefined(installOptions);
```

Comment:
This fails before the changes in this PR

Reply:
May I also suggest updating the test title (here and for the other tests) to reflect this new requirement? Perhaps changing "should call the failure callback..." to "should call the failure callback with installation options..."?

Reply:
Great suggestion -- will update the test label before merging this